### ISSUE TITLE
Cannot read property 'getWrappedInstance' of null

### ISSUE THREAD

**Issue by marktani** [graphcool/graphql-playground#399]
#### This issue pertains to the following package(s):

- [x] GraphQL Playground - Electron App

#### What OS and OS version are you experiencing the issue(s) on?

Mac OS X Sierra

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?

`1.3.20`

#### What is the expected behavior?

Opening another workspace works.

#### What is the actual behavior?

Opening another workspaces returns `Cannot read property 'getWrappedInstance' of null` error

#### What steps may we take to reproduce the behavior?

```sh
npm install -g graphql-cli graphcool@beta
graphql create my-app 
# choose node-advanced
graphcool deploy
yarn start
yarn playground
# open "dev" workspace
```

![image](https://user-images.githubusercontent.com/1780597/34574132-ced36614-f176-11e7-944c-0b33059a1889.png)

**Comment by Huvik** [comment]
Its fixed in #397

**Comment by schickling** [comment]
Thanks a lot for your help @Huvik! 💪
